dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletDestroyDelegateImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.dotmarketing.portlets.contentlet.business;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import java.util.List;
+
+/**
+ * {@inheritDoc}
+ */
+public class ContentletDestroyDelegateImpl implements ContentletDestroyDelegate {
+
+    @WrapInTransaction
+    @Override
+    public void destroy(final List<Contentlet> contentlets, final User user) {
+        try {
+            APILocator.getContentletAPI().destroy(contentlets, user, false);
+        } catch (DotDataException | DotSecurityException e) {
+            Logger.error(this, "Error destroying contents", e);
```

Comment:
print here the contents id at least

Reply:
on the contrary, the destroy method already prints way too much info. That I'd consider removing some of what is actually printed.